docs/project/milestone-1.md

```diff
@@ -148,4 +148,13 @@ users.update(["_id" : "unique_id"],
 
 ### Tasks
 
-You will have to implement the functions marked with `TODO` in `Project/m1/db_conn/app.d`.
+You will have to implement the functions marked with `TODO` in `Project/m1/db_conn/app.d`:
+
+* `addUser - receives a users credentials and adds them to the user database
```

Comment:
Please detail how a password should be stored: hash(gen salt + pass)

Reply:
Didn't we say this is a task for m2?

Reply:
Oh. Ok. It wasn’t clear for me that that was the decision. I’ll merge this then